Docs/source/pages/commands.md

```diff
@@ -2805,12 +2805,12 @@ Sets the value of the static field or property.
 
 **_Parameters_**
 
-| Name          | Type   | Required | Description                                                                                             |
-| ------------- | ------ | -------- | ------------------------------------------------------------------------------------------------------- |
-| componentName | string | Yes      | The name of the component which has the static field or property to be retrieved.                       |
-| propertyName  | string | Yes      | The name of the static field or property to be retrieved.                                               |
-| assembly      | string | Yes      | The name of the assembly the component belongs to.                                                      |
-| updatedProperty | object | Yes      | The new value of the component which has the static field or property to be seted. 
+| Name            | Type   | Required | Description                                                                                                              |
+| --------------- | ------ | -------- | ------------------------------------------------------------------------------------------------------------------------ |
+| componentName   | string | Yes      | The name of the component. If the component has a namespace the format should look like this: "namespace.componentName". |
+| propertyName    | string | Yes      | The name of the property of which value you want to set                                                                  |
+| assemblyName    | string | Yes      | The name of the assembly containing the component. It is NULL by default.                                                |
```

Comment:
You could also update `assembly` to `assemblyName` for `GetStaticProperty `command also (since you changed it here)

![image](https://user-images.githubusercontent.com/62327834/230895371-c3bce64e-27e9-40b9-92cc-a0c59a35ae52.png)